hack/install.sh

```diff
@@ -237,7 +266,7 @@ do_install() {
 			exit 0
 			;;
 
-		fedora|centos)
+		fedora|centos|oracleserver)
```

Comment:
no prob and no rush, its nighttimme haha

On Mon, Jul 27, 2015 at 8:10 PM, Avi Miller <notifications@github.com>
wrote:

> In hack/install.sh
> <https://github.com/docker/docker/pull/14927#discussion_r35611057>:
>
> > @@ -237,7 +266,7 @@ do_install() {
> >  			exit 0
> >  			;;
> >
> > -		fedora|centos)
> > +		fedora|centos|oracleserver)
>
> Ok, I'll fix it up and rebase at the same time. Give me a few minutes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14927/files#r35611057>.
>